<launch>

	<!-- Joy node -->
	<node respawn="true" pkg="joy" type="joy" name="joy" >
		<param name="dev" type="string" value="/dev/input/js0" />
	</node>

	<!-- Axes and scaling -->
	<param name="axis_linear" value="1" type="int"/>
	<param name="axis_angular" value="0" type="int"/>
	<param name="scale_linear" value="0.20" type="double"/>
	<param name="scale_angular" value="0.8" type="double"/>
	
	<!-- At last - teleoperation node -->
	<node pkg="simple_apps" type="piotrek_teleop_node" name="telepiotrek"/>
	<node pkg="simple_apps" type="serialswitch_node" name="serialsw" />

</launch>
